test/_common.py

```diff
@@ -53,7 +53,7 @@
 _item_ident = 0
 
 # OS feature test.
-HAVE_SYMLINK = hasattr(os, 'symlink')
+HAVE_SYMLINK = sys.platform != 'win32'
```

Comment:
yeah, I didn't commit it because i though ti was a bad idea only to do it halfway and not also catch OSError like suggested